.github/CODEOWNERS

```diff
@@ -10,25 +32,14 @@
 /src/promptflow-core/promptflow/executor/ @microsoft/promptflow-execution
 /src/promptflow-core/promptflow/integrations/ @microsoft/promptflow-execution
 /src/promptflow-core/promptflow/storage/ @microsoft/promptflow-execution
-/src/promptflow/tests/executor/ @microsoft/promptflow-execution
 
-/src/promptflow-core/promptflow/core/ @microsoft/promptflow-sdk
-/src/promptflow-devkit/promptflow/_cli/ @microsoft/promptflow-sdk
-/src/promptflow-devkit/promptflow/_sdk/ @microsoft/promptflow-sdk
-/src/promptflow-devkit/promptflow/_proxy/ @microsoft/promptflow-sdk
-/src/promptflow-devkit/promptflow/entities/ @microsoft/promptflow-sdk
-/src/promptflow-devkit/promptflow/operations/ @microsoft/promptflow-sdk
-/src/promptflow-azure/promptflow/azure/ @microsoft/promptflow-sdk
+# 3. Test folder extras
```

Comment:
shall we group 2, 3, 4 together by team ?

Reply:
OK，merge these。